ACadSharp/IO/DXF/DxfStreamWriter/IDxfStreamWriter.cs

```diff
@@ -15,6 +15,8 @@ internal interface IDxfStreamWriter : IDisposable
 		
 		void Write(int code, IVector value, DxfClassMap map = null);
 
+		void WriteCmColor(int code, Color color, DxfClassMap map = null);
```

Comment:
To keep the scope for this task I would suggest to remove this method, if is needed we can implement it in another PR with a dedicated test.

Reply:
OK, I open an issue and PR.